lib/cli.js

```diff
@@ -89,6 +99,17 @@ function run(argv) {
 
   files = expandGlobs(files);
 
+  if (excludedFiles && excludedFiles.length) {
+    excludedFiles = expandGlobs(excludedFiles);
```

Comment:
this will actually be slow if we have a glob that matches too many files (eg. `node_modules` folder) since it will need to expand the glob (can potentially return thousands of files) and check all the files against it...

a better approach would be to pass the `ignore` option to `glob.sync` (that would avoid unnecessary `fs` calls) - see: https://www.npmjs.com/package/glob